test/per_file_coverage.sh

```diff
@@ -88,7 +88,7 @@ declare -a KNOWN_LOW_COVERAGE=(
 "source/extensions/watchdog:83.3" # Death tests within extensions
 "source/extensions/watchdog/profile_action:83.3"
 "source/server:93.3" # flaky: be careful adjusting. See https://github.com/envoyproxy/envoy/issues/15239
-"source/server/admin:97.6"
+"source/server/admin:97.5"
```

Comment:
This is due to I added the new `listenSocketFactories()` and still keep the old `listenSocketFactory()` method, when everything support multiple addresses and no dependence on the `listenSocketFactory()`, then I can remove it, then the coverage will back to the original.

Reply:
Can you add a TODO/comment here about that?

Reply:
got it, thanks